std_msgs/Header header
uint32 can_id
float32 speed
float32 position